container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/logs_collector_functions/implementations/fluentbit/consts.go

```diff
@@ -47,6 +47,15 @@ const (
     DB.sync           normal
     Read_from_Head    true
 
+[FILTER]
```

Comment:
This filter is what requires k8s api server access - https://docs.fluentbit.io/manual/pipeline/filters/kubernetes